### DISCUSSION TITLE
Cutting a Video

### DISCUSSION THREAD

**Discussion by ganzo121** [mifi/lossless-cut#1598] [Q&A]
Hello. I have an issue, as whenever I try to cut a video and make a starting point, it's not accurate because it either starts before or after the exact selected point after exporting, no matter how much seconds I change it stays on the same unselected starting point.

Let me know if I'm doing something wrong. Thanks in advance 🙏

**Comment by mifi** [comment]
https://github.com/mifi/lossless-cut/blob/master/issues.md

**Comment by Kekokaka** [comment]
Or what do you suggest that I try to solve the problem. From what I understood is that the problem is still under review and not solved yet
